x-pack/plugins/alerting/server/task_runner/execution_handler.ts

```diff
@@ -233,7 +233,7 @@ export class ExecutionHandler<
           });
 
           if (isSummaryActionOnInterval(action)) {
-            throttledActions[generateActionHash(action)] = { date: new Date() };
+            throttledActions[action.uuid!] = { date: new Date() };
```

Comment:
We had to add uuid as optional due to some TS problems but actually it's not optional, all the actions has a uuid.